jobs/build/release/Jenkinsfile

```
@@ -207,6 +207,24 @@ node {
                 }
             }
 
+            stage("request upgrade tests") {
+                def (major,minor) = extractMajorMinorVersion(NAME)
+                def previousList = PREVIOUS.trim().tokenize('\t ,')
+                def modeOptions = [ 'aws', 'gcp', 'azure,mirror' ]
+                if ( major == '4' && minor == '1') {
+                    modeOptions = ['aws'] // gcp and azure were not supported in 4.1
+                }
+                def testIndex = 0
+                def testLines = []
+                for ( String from_release : previousList) {
+                    mode = modeOptions[testIndex % modeOptions.size()]
+                    testLines << "test upgrade ${from_release} ${NAME} ${mode}"
+                    testIndex++
+                }
+                def slackChannel = slacklib.to('#team-art')
+                slackChannel.say("Hi @art-team . A new release is ready and needs some upgrade tests to be triggered. Please open a chat with @cluster-bot and issue each of these lines individually:\n${testLines.join('\n')}")
```

If this is done immediately, it would make sense to scope it to the `@release-artists`.
```suggestion
                slackChannel.say("Hi @release-artists . A new release is ready and needs some upgrade tests to be triggered. Please open a chat with @cluster-bot and issue each of these lines individually:\n${testLines.join('\n')}")
```
And also, add it to the `currentBuild.description` (why not?).

---

Also, break up this line, it makes the review impossibly wide :stuck_out_tongue_winking_eye:

---

I originally had release-artist, but I also thought they might struggle with the limit of ten jobs at a time - making it require a bit of team work. But I think you are right, we can start with artist and adapt if it is a problem.  Next push.